src/main/perl/FileWriter.pm

```diff
@@ -117,6 +117,15 @@ File's modification time.
 
 Path for the backup file, if this one has to be re-written.
 
+=item keeps_state
+
+A boolean specifying whether a file change respects the current system
+state or not. A file with C<keeps_state> will be created/modified,
+regardless of any value for C<NoAction>.
```

Comment:
Add: "This is useful when creating temporary files that are required for a NoAction run."